apps/remote_control/lib/lexical/remote_control/search/store/backends/khepri/condition.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Lexical.RemoteControl.Search.Store.Backends.Khepri.Condition do
+  require Record
+
+  for {name, fields} <- Record.extract_all(from_lib: "khepri/include/khepri.hrl") do
```

Comment:
Can we add a moduledoc with a link to docs or something for all the records defined here?

Reply:
your "link" to "docs" is asking a lot. (Khepri uses the erlang docs, and they're _terrible_) I'll see what I can do.